src/components/Support.js

```diff
@@ -0,0 +1,164 @@
+/********************************************************************************
+ * Copyright (C) 2020 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import Nav from './Nav'
+import React from 'react'
+import { breakpoints } from '../utils/variables'
+import styled from '@emotion/styled'
+import EclipseSourceLogo from '../resources/eclipse-source.svg'
+import TypeFoxLogo from '../resources/typefox.png'
+import Background from '../resources/background-image.png'
+
+const StyledSupport = styled.div`
+.header {
+    position: relative;
+    background-image: url(${Background});
+    background-size: cover;
+    padding-bottom: 3rem;
+    border-bottom: 10px solid #f8f8f8;
+    @media(max-width: ${breakpoints.xmd}) {
+        padding-top: 15rem;
+    }
+    &__logo-box {
+        margin-bottom: 3rem;
+    }
+    &__logo {
+        height: 3rem;
+    }
+    .btn {
+        max-width: 20rem;
+        @media(max-width: 385px) {
+            &:not(:last-child) {
+                margin-right: 0;
+            }
+        }
+        @media(max-width: 800px) {
+            &:not(:last-child) {
+                margin-right: 2rem;
+            }
+        }
+    }
+    &__buttons {
+        display:flex;
+        justify-content: space-evenly;
+        align-items: center;
+        text-align: center;
+        padding: 3rem 10vw;
+        @media(max-width: 800px){
+            padding: 3rem 0;
+        }
+        
+    }
+}
+h1 {
+    margin-bottom: 1rem;
+    font-size: 3.4rem;
+}
+.support {
+    padding: 3rem 0;
+}
+
+.support__container {
+    display: flex;
+
+    @media(max-width: ${breakpoints.md}) {
+        flex-direction: column;
+    }
+}
+.reverse {
+    flex-direction: row-reverse;
+    @media(max-width: ${breakpoints.md}) {
+        flex-direction: column-reverse;
+    }
+}
+.supporter {
+    flex: 1;
+    margin: 2rem 0;
+
+    & > div {
+        text-align: center;
+    }
+
+    h3 {
+        margin-top: 2rem;
+    }
+
+    & > p {
+        margin: 0 auto;
+        margin-bottom: 1rem;
+        max-width: 45rem;
+    }
+
+    img {
+        height: 7rem;
+    }
+}
+`
+
+const supporters = [
+    {
+        img: EclipseSourceLogo,
+        title: 'EclipseSource',
+        link: 'https://eclipsesource.com/technology/eclipse-theia/',
```

Comment:
Shouldn't it point to https://eclipsesource.com/?
It is currently the same link as `support`.